### PR TITLE
Some simple initial tests for MediaView

### PR DESCRIPTION
**Pull request by hughns**
The purpose of moving videoMuted from CSS class to inline style is to make it more testable. I'm open to a more elegant way!

### CODE REVIEW

File: src/tile/MediaView.test.tsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+          encryptionStatus={EncryptionStatus.Connecting}
+          mirror={false}
+          unencryptedWarning={false}
+          video={trackReference}
+          member={undefined}
+        />,
+      );
+      expect(screen.getByTestId("name_tag")).toHaveTextContent("Bob");
+    });
+  });
```
Comment (toger5) [inline src/tile/MediaView.test.tsx]: Can we not combine those two tests?

Comment (hughns) [reply src/tile/MediaView.test.tsx]: Whilst the code is very similar and the expectations could be combined, I would argue that the purpose of each test case is clearer in the way that it is currently written.

If you are concerned about code duplication I think I can do something about that whilst keeping things separate. I will push an extra commit.

**Comment by hughns** [comment]
I have tried to make the intent of each test case clearer in the latest commit by factoring out the common base props for each test case.

File: src/tile/MediaView.test.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  test("is accessible", async () => {
+    const { container } = render(<MediaView {...baseProps} />);
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  describe("placeholder track", () => {
+    test("neither video nor avatar are shown", () => {
+      render(<MediaView {...baseProps} video={trackReferencePlaceholder} />);
+      expect(screen.queryByTestId("video")).toBeNull();
+      expect(screen.getByTestId("avatar")).not.toBeVisible();
```
Comment (robintown) [inline src/tile/MediaView.test.tsx]: Can we use role-based queries for these rather than test IDs?

File: src/tile/MediaView.test.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+  describe("unencryptedWarning", () => {
+    test("is shown and accessible", async () => {
+      const { container } = render(
+        <TooltipProvider>
+          <MediaView {...baseProps} unencryptedWarning={true} />
+        </TooltipProvider>,
+      );
+      expect(await axe(container)).toHaveNoViolations();
+      expect(screen.getByTestId("unencrypted_warning_icon")).toBeTruthy();
```
Comment (robintown) [inline src/tile/MediaView.test.tsx]: Could also use a role or label query here

File: src/tile/MediaView.test.tsx (hunk 2 above)
Comment (hughns) [reply src/tile/MediaView.test.tsx]: I don't see a role for video elements so I am leaving that as a testid.

**Review by toger5 — APPROVED** [review]
The video with the test id makes sense to me.
I also did a quick search and nothing obvious came up.

**Pull request merged**